index.html

```diff
@@ -0,0 +1,181 @@
+          <h3 class="card-title">Web Development</h3>
+          <p class="card-content">We will help you to learn the latest top frontend technologies. Such as: HTML5, CSS3,
+            BootStrap5, JS, React
+            and more. </p>
+        </article>
+
+        <article class="card">
+          <img class="card-img" src="./Images/two.png" alt="second card image">
+          <h3 class="card-title">Web Hosting</h3>
+          <p class="card-content">Currently we are hosting 2000+ web applications. Our goal is to provide outstanding
+            services for the best possible price.</p>
+        </article>
+
+        <article class="card">
+          <img class="card-img" src="./Images/three.png" alt="third card image">
+          <h3 class="card-title">Domain Registration</h3>
+          <p class="card-content">Online success starts with a great domain. At FR Graphics, we make it easy and
+            inexpensive to get the domain you want, secure and fast! </p>
+        </article>
+
+        <article class="card">
+          <img class="card-img" src="./Images/four.png" alt="fourth card image">
+          <h3 class="card-title">Digital Branding</h3>
+          <p class="card-content">To stand out, brands must have personality. They must express themselves in a unique
+            way to be easily recognizable. We work with you to define your brand's. </p>
+        </article>
+
+        <article class="card">
+          <img class="card-img" src="./Images/five.png" alt="fifth card image">
+          <h3 class="card-title">Network Management</h3>
+          <p class="card-content">We review the components of your existing network and make sound recommendations for
+            improvements to the network's design and performance. </p>
+        </article>
+      </section>
+
+      <article class="card-btn">
+        <button class="mob-btn" type="button">Join Our Community</button>
+
+        <a href="#" class="desk-btn">Check Our Curriculum</a>
+      </article>
+    </section>
+  </main>
+  <!-- Main section End -->
+
+  <!-- Our Team section -->
+  <section class="team-container" id="team-container">
+    <h2 class="team-title">Our Team Members</h2>
+    <div class="team-content" id="team-content">
+      <!-- Data Loading DYNAMICALLY -->
+    </div>
+  </section>
+
+  <!-- Sponsors Section Start -->
+  <section id="supporters" class="sponsors">
+
+    <article class="title-container">
+      <h2 class="s-title">Our Sponsors</h2>
+      <div class="line extend-line"></div>
+    </article>
+
+    <article class="sponsors-logos">
+      <img class="s-logo" src="./Images/Amazon.png" alt="amazon logo">
+      <img class="s-logo" src="./Images/Google.png" alt="google logo">
+      <img class="s-logo" src="./Images/Netflix.png" alt="netflix logo">
+      <img class="s-logo" src="./Images/PayPal.png" alt="paypal logo">
+      <img class="s-logo" src="./Images/Spotify.png" alt="spotify logo">
+    </article>
```

Comment:
Wow! it looks awesome, but it's required to follow the original design as much as possible, so please make sure to use the same background color for this section.
Yours:
![image](https://user-images.githubusercontent.com/34779928/205144018-9668ae14-5675-48f8-8b2d-93b15d215e01.png)

Original:
![image](https://user-images.githubusercontent.com/34779928/205144065-f8874e4f-1367-4782-9091-f35f8214dbda.png)